packages/cozy-mespapiers-lib/src/components/ModelSteps/widgets/InputTextAdapter.jsx

```diff
@@ -160,8 +160,8 @@ const InputTextAdapter = ({
       helperText={helperText}
       value={currentValue}
       inputProps={{
-        maxLength: maxLength,
-        minLength: minLength,
+        maxLength: maxLength >= 0 ? maxLength : undefined,
```

Comment:
```suggestion
        maxLength: maxLength >= 0 && maxLength,
```

Reply:
There is a diff, before we were returning `undefined`, with your suggested change we now return `false`